trendico/models.py

```diff
@@ -1,3 +1,48 @@
 from django.db import models
-
+from datetime import datetime, timedelta
 # Create your models here.
+CATEGORY_CHOICES = (
+    ('L', 'Laptop'),
+    ('T', 'Tab'),
+    ('C', 'Camera'),
+    ('H', 'Headphone')
```

Comment:
Usually, we use 0,1,2,3 ... rather than L, T, C, H for this kind of choices

Reply:
but rather than make it choices, make a ProductCategory model so categories can be configurable from admin